README.md

```diff
@@ -1,6 +1,6 @@
-# ArcGIS Maps SDK for Swift Samples
+# ArcGIS Maps SDK for Swift Samples [![](https://user-images.githubusercontent.com/2257493/54144188-6fe0fc00-43e8-11e9-8cf5-229af80f604a.png)](https://apps.apple.com/us/app/arcgis-maps-swift-samples/id1630449018)
```

Comment:
Can you download the AppStore badge as an svg from Apple's marketing [guidelines](https://developer.apple.com/app-store/marketing/guidelines/) or use the [link](https://developer.apple.com/assets/elements/badges/download-on-the-app-store.svg) directly?

Reply:
Updated